pkg/services/store/test-svgs/README.md

```diff
@@ -0,0 +1,5 @@
+Sources
```

Comment:
The files in this folder don't seem used.

Reply:
removed